pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -245,6 +246,21 @@ func (a *ApplicationBackupController) handle(ctx context.Context, backup *stork_
 		return nil
 	}
 
+	if labelSelector := backup.Spec.NamespaceSelector; len(labelSelector) != 0 {
+		namespaces, err := core.Instance().ListNamespaces(labelSelector)
+		if err != nil {
```

Comment:
Instead of returning error, it is better to record the error as event in the applicationbackup CR as it is done in other places.

Reply:
ok